frontend/src/components/UserCaloriesPage.js

```diff
@@ -243,6 +245,55 @@ function UserCaloriesPage(props) {
   console.log(todayCaloriesBurned)
   console.log(todayCaloriesConsumed)
 
+  const history = useHistory();
+
+const redirectToEventWithModalOpen = (eventName) => {
+  history.push({
+    pathname: '/Events',
+    state: { openModalForEvent: eventName }
+  });
+};
+
+useEffect(() => {
+
+  axios.get('/api/enrolled-events', {
+    headers: {
+      Authorization: "Bearer " + props.state.token,
+    }
+  })
+  .then(response => {
+    setEnrolledEvents(response.data); 
+  })
+  .catch(error => {
+    console.error("Error fetching enrolled events:", error);
+  });
+}, []);
```

Comment:
Please delete this. The expected functionality is already being done. The route specified is not mapped in the backend as well, rendering this code-block unused.

Reply:
Deleted the redundant code.